src/Illuminate/Foundation/Configuration/Middleware.php

```diff
@@ -451,6 +451,7 @@ public function appendToPriorityList($after, $append)
     public function getGlobalMiddleware()
     {
         $middleware = $this->global ?: array_values(array_filter([
+            \Illuminate\Foundation\Http\Middleware\ValidateUTF8Path::class,
```

Comment:
https://github.com/laravel/framework/blob/2d10f2aeddc94e57529a2fb33a50bed8810e3923/src/Illuminate/Support/Facades/Date.php#L74

We don't have any existing class that contains the word `UTF8` but it might be best to make it consistent with existing code. `ValidateUtf8Path` or `ValidateEncodedPath`

Reply:
Good idea. I changed it to `ValidatePathEncoding`.